src/modules/actions.js

```diff
@@ -33,7 +33,8 @@ export interface CurrencyEngineChangedTxs {
   type: 'CURRENCY_ENGINE_CHANGED_TXS';
   payload: {
     txs: Array<any>,
-    walletId: string
+    walletId: string,
+    txidHashes: any
```

Comment:
I have a feeling this will need to be changed to something like `Array<string>`?